.gitignore

```diff
@@ -5,3 +5,440 @@ __pycache__
 gatorgrouper_django/gatorgrouper/migrations/
 /.coverage
 secret_key.py
+
+# Created by https://www.gitignore.io/api/vim,git,macos,linux,django,python,eclipse,windows,intellij,notepadpp
+# Edit at https://www.gitignore.io/?templates=vim,git,macos,linux,django,python,eclipse,windows,intellij,notepadpp
+
+### Django ###
+*.log
+*.pot
+*.pyc
+__pycache__/
+local_settings.py
+db.sqlite3
```

Comment:
We should ignore *all* `sqlite3` file!

Reply:
@Michionlion 
I want to suggest that we actually leave the db in the repo, but if you think otherwise then there are a couple of things that should end up getting documented somewhere.  You cannot run django normally if you do not have the db without first running the commands `python3 manage.py makemigrations` and `python3 manage.py migrate` to create it and then running `python3 manage.py createsuperuser` to be able to access the admin pages.  All of the models and logins get added to the database and if you don't have it in the repo then they must get recreated as part of the setup.

When this eventually goes onto AWS, the database does need uploaded there as the site will not correctly function without it.  There is a way to have it so that when you push to master the site will automatically update, but this will not work correctly unless the db is committed.

The only reason that I can think of not to commit the database is for security reasons, but the development version of the tool shouldn't have any sensitive information.

Reply:
Okay, that makes sense. I have seen merge conflicts with the database as well however, which may be complicated for students to resolve, since they will need to specify `--ours` or `--theirs` for the database file.

Reply:
@Alex-Yarkosky Can you remove the `sqlite3` ignore?